cmd/panfs-multipart.go

```diff
@@ -248,7 +248,7 @@ func (fs *PANFSObjects) NewMultipartUpload(ctx context.Context, bucket, object s
 	uploadID := mustGetUUID()
 	uploadIDDir := fs.getUploadIDDir(bucketPath, bucket, object, uploadID)
 
-	err = mkdirAll(uploadIDDir, 0o755)
+	err = panMkdirAll(uploadIDDir, fs.defaultDirMode, fs.defaultOwner, fs.defaultGroup)
```

Comment:
is upload directory located in .s3 and is it an internal MinIO folder to keep upload parts while upload is in progress? If, yes, I'd avoid setting non-root rights for the hierarchy.

Reply:
Rolled back